app/controllers/api/v1/inventory_numbers_controller.rb

```diff
@@ -19,7 +19,9 @@ def create
 
       api :PUT, "/v1/inventory_numbers", "Delete inventory_number"
       def remove_number
-        InventoryNumber.find_by(code: params[:code]).destroy
+        if (inventory_number = InventoryNumber.find_by(code: params[:code]))
```

Comment:
@namrataukirde can we use safe navigation here to find it and destroy it ?
Something like `InventoryNumber.find_by(code: params[:code])&.destroy)`
or `InventoryNumber.find_by(code: params[:code]).try(:destroy)`